components/br-stream/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "br-stream"
+version = "0.1.0"
+edition = "2018"
+
+[[bin]]
+name = "playground"
+path = "bin/playground.rs"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+# [features]
+# default = ["protobuf-codec"]
+# protobuf-codec = ["kvproto/protobuf-codec"]
+# prost-codec = ["prost", "kvproto/protobuf"]
+
+[dependencies]
+
+tokio = { version = "1.5", features = ["rt-multi-thread", "macros", "time", "sync"] }
+async-trait = { version = "0.1" }
+thiserror = "1"
+bytes = "0.4"
+etcd-client = { version = "0.7", features = ["pub-response-field"] }
+kvproto = { git = "https://github.com/pingcap/kvproto.git", branch = "br-stream"}
```

Comment:
i think you need to change every `Cargo.toml` importing `kvproto` to use the branch `br-stream` too. otherwise we'll have got two different `kvproto`s.

Reply:
thanks to 3pointer...

BTW, I guess get two versions of `kvproto` wouldn't bring bad things? (Hopefully...)

Reply:
well the types from the two `kvproto` are not considered the same so e.g. you can't pass the `StorageBackend` from one to the other directly